unit_07/hypothesis_test_practice_activity.Rmd

```diff
@@ -1,24 +1,41 @@
 ---
-title: "hypothesis_test_practice_activity"
+title: "Hypothesis Test Practice Activity"
 author: "Alex Hughes"
 date: "10/4/2021"
-output: html_document
+output: bookdown::gitbook
 ---
 
+# Hypothesis Test Practice Activity
+
+In this short activity, you're going to write, and execute a short series of hypothesis tests using the `R` estimating framework. 
+
+The data that you will be using comes from *Tidy Tuesday* and is about the costs of university education for undergraduates. 
+
+This data collection effort is described in detail [on Github](https://github.com/rfordatascience/tidytuesday/blob/master/data/2020/2020-03-10/readme.md#tuition_costcsv), but includes the following tables: 
+
+- `tuition_cost`
+- `tuition_income` 
+- `historial_tuition` 
+- `salary_potential` 
+- `diversity_school` 
+
+We are going to ask as series of questions that can be answered with the constrained set of tests that we have available to use from the course. 
+
+For each of the questions, the data *is* available, but you might have to join a table or two, recode a variable or two, or otherwise do a little bit of data work to get the data ready to run the test. 
+
+For each test that you conduct, please (a) evaluate the assumptions of the test to see if the data satisfy these assumptions; (b) state the null hypothesis that is being evaluated; (c) state the criteria that would lead you to reject the null hypothesis; (d) conduct and interpret the test; and (e) tell us whether the difference you observe between the groups is an *important* one. 
```

Comment:
"state the criteria that would lead you to reject the null hypothesis" is it valid for a student to write "p<.05" for every test? perhaps that's a reason to remove this component and test it more directly another way.